qiita_db/test/test_data.py

```diff
@@ -785,6 +785,21 @@ def test_create(self):
         # preprocessed_data_id, processed_Data_id
         self.assertEqual(obs, [[1, 2]])
 
+    def test_delete(self):
+        """Correctly deletes a processed data and raises an error if it has
```

Comment:
In theory, these descriptions have to be one-liners, otherwise they do not show up correctly (or at least they did not show up before...)